ui/app/AppLayouts/Onboarding/OnboardingLayout.qml

```diff
@@ -35,6 +35,10 @@ OnboardingBasePage {
         id: loader
         anchors.fill: parent
         sourceComponent: {
+            // If metrics is enabled, we send a metric to the screen visited
+            // TODO check if enabled
+            Global.addCentralizedMetricIfEnabled(Constants.navigationMetric, {viewId: root.startupStore.currentStartupState.stateType})
+
             switch (root.startupStore.currentStartupState.stateType) {
```

Comment:
I think we could just share the `stateType` and `flowType` right from Nim:
https://github.com/status-im/status-desktop/blob/e03ccbfe2159eebc9c741f5454b8b6bafb17433f/src/app/modules/startup/view.nim#L76-L77

It would be a single place, but much more info of how the user goes through onboarding.